packages/luigis-box/src/Component/LuigisBox/LuigisBoxClient.php

```diff
@@ -102,6 +102,8 @@ public function getData(
                 [
                     'exception' => $e,
                     'luigisBoxBatchLoadData' => $luigisBoxBatchLoadData,
+                    'options' => $options,
+                    'response' => isset($response) ? $response->getContent(false) : null,
```

Comment:
just a thought - would not `$response->toArray(false)` provide better readable information?